### ISSUE TITLE
--curl should escape ampersands

### ISSUE THREAD

**Issue by dweaver** [exosite/exoline#124]
The ampersand should be escaped in the --curl output, so it can be copied and pasted unmodified.

```
$ exo --curl sn list metrobus
DEBUG:pyonep.provision: curl https://m2.exosite.com:443/provision/manage/model/metrobus/?limit=1000&offset=0 -X GET -m 5 -H 'Accept: text/plain, text/csv, application/x-www-form-urlencoded' -H 'X-Exosite-Token: 30c...'
```

**Comment by danslimmon** [comment]
Will be resolved by https://github.com/exosite-labs/pyonep/pull/15

**Comment by dweaver** [comment]
included in 0.12.0

**Comment by dweaver** [comment]
Oops, that'd be pyonep that is released, so this isn't closed yet.

**Comment by dweaver** [comment]
Closed in 0.9.19
